src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -314,5 +327,19 @@ private static void WriteResultTable(IEnumerable<Counter> counters, Logger logge
                 logger.Log($"{counter.Name,-15}|{average,-15}|{min,-15}|{max,-15}");
             }
         }
+
+        private static Dictionary<string, string> ParseStringToDictionary(string s)
+        {
+            Dictionary<string, string> dict = new Dictionary<string, string>();
+            foreach (string substring in s.Split(';', StringSplitOptions.RemoveEmptyEntries))
+            {
+                string key = substring.Split('=').First();
+                string value = substring.Split('=').Last();
+                if(!String.IsNullOrWhiteSpace(key) && !String.IsNullOrWhiteSpace(value)){
```

Comment:
I would not require a value here. `SET FOO=` is valid to unset `FOO`. Instead of the value is null, set it to empty string?

Reply:
But do we want that semantic enforced here? It feels like we want to use this just to set new environment variables for when we want to run, not use it to arbitrarily shape the environment space

Reply:
I can pretty easily imagine I want to run scenarios where I ensure a particular variable isn't set. Say I want to be very sure tiered compilation isn't disabled, etc.

However the semantics don't quite work like I thought they would. To make this work we'd have to actually remove the variable from the collection if it had an empty string. What we have here is sufficient.

Reply:
great solution below!